collectd/packages.sls

```diff
@@ -0,0 +1,13 @@
+{% from "collectd/map.jinja" import collectd_settings with context %}
+
+include:
+  - collectd
```

Comment:
Then you also need a watch/listen somewhere. Merely including a file doesn't automatically link it to something.